java/parquet/src/main/java/sleeper/utils/HadoopConfigurationProvider.java

```diff
@@ -33,6 +34,16 @@ public class HadoopConfigurationProvider {
     private HadoopConfigurationProvider() {
     }
 
+    public static Configuration getConfigurationForClient() {
+        Configuration conf = new Configuration();
```

Comment:
Should we include the S3A configuration properties here?

Reply:
Yes, we need any of the properties that are relevant to queries.

Reply:
Added missing S3A config